src/screens/ConfirmationContainer/index.js

```diff
@@ -29,12 +29,13 @@ type Props = {
   amount: String,
   estimate: String,
   purpose: String,
-  app: String
+  app: String,
```

Comment:
Please use lowercase `string` instead. It seems that it was incorrectly written on the old code, so please change it as well.

Reply:
Done.